omero/developers/Python.rst

```diff
@@ -6,24 +6,57 @@ libraries.
 
 From OMERO 5.6.0 release, the client library ``omero-py`` supports Python 3 and
 is now available on PyPI_ and Conda_. The ``omero-py`` API documentation is available at https://omero-py.readthedocs.io/.
-We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv`` or ``conda`` (preferred).
-If you opt for Conda_, you will need
-to install it first, see `miniconda <https://docs.conda.io/en/latest/miniconda.html>`_ for more details.
-
-To install ``omero-py`` using venv:
+We recommend you use a Python virtual environment to install the client library. You can create one using either ``venv``, ``conda`` or ``mamba`` .
+Before installing ``omero-py``, we recommend to install the `Zeroc IcePy 3.6 <https://zeroc.com/ice/downloads/3.6>`_ Python bindings.
+
+Our commercial partner `Glencoe Software <https://www.glencoesoftware.com/>`_ has produced several Python wheels to install the Ice-Python bindings depending on the desired Python version and the operating system.
+
+.. list-table::
+    :header-rows: 1
+    :align: left
+
+    * - OS
+      - Python version supported
+      - Ice-Python Binding
+    * - | RHEL 8, RHEL 9, RockyLinux 9
+        | Ubuntu 20.04, and Ubuntu 22.04
+      - 3.8, 3.9, 3.10, 3.11 and 3.12
+      - `zeroc-ice-py-linux-x86_64 <https://github.com/glencoesoftware/zeroc-ice-py-linux-x86_64/releases/tag/20240202>`_
+    * - Windows
+      - 3.8, 3.9, 3.10, 3.11 an 3.12
+      - `zeroc-ice-py-win-x86_64 <https://github.com/glencoesoftware/zeroc-ice-py-win-x86_64/releases/tag/20240325>`_
+    * - macOS 11.0 or later (``x86_64``)
+      - 3.8 and 3.9
+      - `zeroc-ice-py-macos-x86_64 <https://github.com/glencoesoftware/zeroc-ice-py-macos-x86_64/releases/tag/20231130>`_
+    * - macOS ``x86_64`` and ``arm64``
+      - 3.10, 3.11 and 3.12
+      - `zeroc-ice-py-macos-universal2 <https://github.com/glencoesoftware/zeroc-ice-py-macos-universal2/releases/tag/20240131>`_
+
+
+For example, to install ``omero-py`` using ``venv`` with Python 3.11 on Ubuntu 22.04:
```

Comment:
Maybe something like:  "...to install omero-py using pip in a venv environment created with Python3.11... "
(since we are not specifying Python 3.11 here)